pkgs/by-name/ju/junest/package.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, stdenv, fetchFromGitHub, wget }:
+
+stdenv.mkDerivation rec {
+  version = "7.4.8";
+  pname = "junest";
+  src = fetchFromGitHub {
+    owner = "fsquillace";
+    repo = "junest";
+    rev = "refs/tags/${version}";
+    hash = "sha256-9yrQ721fHUxXEZ0mh27SB8yoUH67ltOktUq3kr4qrBc=";
+  };
+
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mkdir -p $out/lib
+    cp -r $src/bin/ $out/
+    cp -r $src/lib/ $out/
+  '';
+
+  postBuild = ''
```

Comment:
because this is postBuild. it gets run before install, install overwrites `$out/bin/junest`

Reply:
Fixed!